devpod-wrapper

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# If we're outside the flatpak, we will run the flatpak command
+if [ -z "${FLATPAK_ID}" ]; then
+	flatpak run --branch=master --arch=x86_64 --command=devpod-cli sh.loft.Devpod "$@"
```

Comment:
why is this needed? 

Also there is no `master` branch for the flatpak when user install it from flathub.

Reply:
Devpod has 2 components, the CLI and the desktop app. The desktop app calls the CLI for most of it's functionality. We wanted to ensure that if someone used the devpod CLI it would always use the same version as the desktop app, even if they didn't use `flatpak run ...` and just executed `devpod` in there PATH. I will remove the branch master :+1: